src/legacy/core_plugins/kbn_doc_views/public/views/table/table_row_btn_collapse.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiToolTip } from '@elastic/eui';
+
+export interface Props {
+  onClick: () => void;
+  isCollapsed: boolean;
+}
+
+export function DocViewTableRowBtnCollapse({ onClick, isCollapsed }: Props) {
+  return (
+    <EuiToolTip
+      content={
+        <FormattedMessage
+          id="kbnDocViews.table.toggleFieldDetails"
+          defaultMessage="Toggle field details"
+        />
+      }
+    >
+      <button
+        aria-expanded={!isCollapsed}
+        aria-label="Toggle field details"
+        className="discover-table-open-button kbnDocViewer__collapseButton"
+        data-test-subj="collapseBtn"
+        onClick={() => onClick()}
+      >
+        <span className={`kuiIcon ${!isCollapsed ? 'fa-caret-down' : 'fa-caret-right'}`}></span>
```

Comment:
Definitily, that's why I started #43577 , is it ok for you to change it in the next step? because it's not only changing the icons, there are more things like alignment, height, border, margin to adapt

Reply:
I suppose so, though icon swapping seems like a quick win for EUI conversion and doesn't seem necessary to be wrapped up in an entire re-design. We'd **really** like to get rid of all our FA dependencies as soon as we can. If you can do the replacement, we can fix up any alignment issues that might come with it. But I can't imagine it would take more than an hour.

Reply:
ok. i see what I can do, but it should be finished on Monday (there is another PR) waiting for it. Today it's already a bit late. maybe you could help me selecting icons, the magnifier icons are clear, but what to use for *adding a column* and the *exists* filter? many thanks

Reply:
columns? 
![Bildschirmfoto 2019-08-23 um 17 04 01](https://user-images.githubusercontent.com/463851/63602572-09335180-c5c8-11e9-9d20-8120c2f492ae.png)

Reply:
exists? (that's not easy to picture, just taking a similar icon)

![Bildschirmfoto 2019-08-23 um 17 05 47](https://user-images.githubusercontent.com/463851/63602680-4b5c9300-c5c8-11e9-9871-466d4423e6fc.png)

Reply:
Given that the *columns* icon toggles (signifying adds and removes), the `tableOfContents` icon feels appropriate and it also resembles the current fontawesome icon. 

The *exists* icon only adds so we could use the `indexOpen` icon which has a _plus_ as part of the design. Another alternative here, based upon literal usage, would be the `filter` icon.

<img width="238" alt="Screenshot 2019-08-23 10 19 04" src="https://user-images.githubusercontent.com/446285/63604198-cdc96080-c590-11e9-82a4-8f270458071a.png">

<img width="237" alt="Screenshot 2019-08-23 10 22 09" src="https://user-images.githubusercontent.com/446285/63604211-d1f57e00-c590-11e9-8b64-8ba67eb705d0.png">

Reply:
what do you think? <img width="532" alt="Bildschirmfoto 2019-08-23 um 18 48 55" src="https://user-images.githubusercontent.com/463851/63609122-cc228b80-c5d6-11e9-976b-64f38eaa28d6.png">

Reply:
Looks good, thanks for cleaning this up!